calendar-webapp/src/main/java/org/exoplatform/calendar/webui/UICalendarPortlet.java

```
@@ -454,4 +451,19 @@ public void createActionConfirms(UIConfirmation uiConfirmation) {
     actionConfirms.add(new UIConfirmation.ActionConfirm("AbortClose", no));
     uiConfirmation.setActions(actionConfirms);
   }
+
+  /**
+   * use in template
+   */
+  public int getTimeShift(){
+    int defaultTime = 2;
+    try {
+      defaultTime = Integer.parseInt(PropertyManager.getProperty("exo.calendar.default.event.suggest"));
+    } catch (NumberFormatException e) {
+      log.error("exo.calendar.default.event.suggest must be int, it currently is: {}", defaultTime);
+      defaultTime = 1;

```

Why defaultTime is re-assigned to 1 ? which will mislead the error log message in the previous line.

---

i'm done fixing this